lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -11,11 +11,13 @@ def self.run(params)
         # Checkout develop and update
         Fastlane::Helper::GitHelper.checkout_and_pull('develop')
 
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
```

Comment:
Ruby offers a common syntax for this:
```suggestion
        app = params[:app] || ENV['PROJECT_NAME']
```

Though to be honest you could just go with this, because the fallback to read the ENV var is already handled by Fastlane's `ConfigItem`:
```suggestion
        app = params[:app]
```